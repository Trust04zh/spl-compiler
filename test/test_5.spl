int mei_huo_zheng_le(int a, float b, char c) {
    return a;
}

char yao_da_huo_ji() {
    return 'a';
}

int act(char cc) {
    return 0;
}

int main() {
    mei_huo_zheng_le(1, 2.0, 'c');
    mei_huo_zheng_le(1, 2.0, 'c', d);
    mei_huo_zheng_le(act(yao_da_huo_ji()), 2.0, yao_da_huo_ji(), 'd');
    mei_huo_zheng_le(1, 2.0);
    return 0.0;
}
