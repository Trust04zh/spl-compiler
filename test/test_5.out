Error type 1 at Line 15: variable d is used without definition
Error type 9 at Line 16: a function's arguments mismatch the declared parameters (either types or numbers, or both)
Error type 9 at Line 17: a function's arguments mismatch the declared parameters (either types or numbers, or both)
Error type 8 at Line 18: a function's return value type mismatches the declared type
